valhalla/baldr/graphtile.h

```diff
@@ -315,7 +315,7 @@ class GraphTile {
 
   // Graph tile memory, this must be shared so that we can put it into cache
   // Apparently you can std::move a non-copyable
-  boost::shared_array<char> graphtile_;
+  boost::shared_ptr<std::vector<char>> graphtile_;
```

Comment:
i think we can only do that if we make `graphtile` non copyable which might be a good idea. maybe i'll try that later after we get this merged!